community-tutorials/install-docker/01-en.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Setup Docker on a Ubuntu server
```

Comment:
Please specify more precisely what you are addressing in this tutorial (docker-compose, minecraft-server) for title, slug, and description

Reply:
Adjusted the title, description and slug to be more precise, thank you for the feedback 👍